hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -150,6 +150,7 @@ public class HoodieWriteConfig extends HoodieConfig {
       .defaultValue(HoodieTableVersion.current().versionCode())
       .withValidValues(
           String.valueOf(HoodieTableVersion.SIX.versionCode()),
+          String.valueOf(HoodieTableVersion.SEVEN.versionCode()),
```

Comment:
@nsivabalan @yihua Had to make this change as noticed test failures.

https://gist.github.com/rahil-c/e7fbdb6a846c0f3cd1130aeb6b96ea78

Reply:
I will see if there is any other thing i can try since im still concerned that if a user configures this value of 7, this technically does not map to an official release.

Reply:
Right, `HoodieTableVersion.SEVEN.versionCode()` should be disallowed as the `hoodie.write.table.version`.